toolkit/resources/manifests/package/pkggen_core_aarch64.txt

```diff
@@ -1,5 +1,5 @@
 filesystem-1.1-8.cm2.aarch64.rpm
-kernel-headers-5.15.2.1-5.cm2.noarch.rpm
+kernel-headers-5.15.2.1-6.cm2.noarch.rpm
```

Comment:
You're going to need to rebase onto the recently merged kernel update https://github.com/microsoft/CBL-Mariner/pull/2104